### ISSUE TITLE
Cannot install Eclipse plugin

### ISSUE THREAD

**Issue by mlevin2** [ContextMapper/context-mapper-dsl#349]
**Describe the bug**
"One or more items could not be found" error

**To Reproduce**
1. Attempt to install Context Mapper plugin from Eclipse Marketplace

**Expected behavior**
All components would install.

**Screenshots / Exceptions**
```
Cannot complete the install because one or more required items could not be found.
  Software being installed: Context Mapper DSL Editor 6.9.0 (org.contextmapper.dsl.feature.feature.group 6.9.0)
  Missing requirement: org.contextmapper.dsl 6.9.0 (org.contextmapper.dsl 6.9.0) requires 'osgi.bundle; org.slf4j.api 0.0.0' but it could not be found
  Cannot satisfy dependency:
    From: Context Mapper DSL Editor 6.9.0 (org.contextmapper.dsl.feature.feature.group 6.9.0)
    To: org.eclipse.equinox.p2.iu; org.contextmapper.dsl [6.9.0,6.9.0]
```

**Input files to reproduce**

**IDE and Plugin (please complete the following information):**
 - Eclipse version 4.29.0
 - ContextMapper plugin version 6.9.0 

**Additional context**

**Comment by stefan-ka** [comment]
@mlevin2, it could take some time until we are able to fix this. Alternatively, you could use Eclipse 2023-06.

**Comment by stefan-ka** [comment]
Hi @mlevin2!

We just released Context Mapper v6.10.0 which is now compatible with Eclipse 2023-09. So I close this issue. Let me know if you still have troubles.

Best regards,
Stefan
